src/Components/NewForm.vue

```diff
@@ -159,6 +207,8 @@ export default {
 					this.createNotifications,
 					this.sendEmails,
 					this.allowComments,
+					new Date(this.scheduleTime).getTime() / 1000, // time in seconds
+					new Date(this.deleteTime).getTime() / 1000,
```

Comment:
```js
new Date(null).getTime()
0 
```

So you are never sending null.
But also just a detail as in the Controller and Manager null and 0 are treated the same.